src/api/apiKey.js

```diff
@@ -0,0 +1,3 @@
+const apiKey = '251e77f3'
```

Comment:
it is common to use uppercase + underscores for such constants
```suggestion
const API_KEY = '251e77f3'
```